src/SFA.DAS.Commitments.Api/Orchestrators/Mappers/CommitmentMapper.cs

```diff
@@ -83,6 +83,7 @@ public CommitmentView MapFrom(Commitment commitment, CallerType callerType)
                     EmployerAccountId = commitment.EmployerAccountId,
                     LegalEntityId = commitment.LegalEntityId,
                     LegalEntityName = commitment.LegalEntityName,
+                    LegalEntityAddress = commitment.LegalEntityAddress,
```

Comment:
Ideally we'd add a unit test to test the mapping of the new field, but it's a simple change so it's not critical.